frontend/public/components/secret.jsx

```diff
@@ -84,21 +84,48 @@ const SecretDetails = ({obj: secret}) => {
 const SecretsList = props => <List {...props} Header={SecretHeader} Row={SecretRow} />;
 SecretsList.displayName = 'SecretsList';
 
-const secretType = secret => secret.type;
+const IMAGE_FILTER_VALUE = 'Image';
+const SOURCE_FILTER_VALUE = 'Source';
+const TLS_FILTER_VALUE = 'TLS';
+const SA_TOKEN_FILTER_VALUE = 'Service Account Token';
+const OPAQUE_FILTER_VALUE = 'Opaque';
+
+const secretTypeFilterValues = [
+  IMAGE_FILTER_VALUE,
+  SOURCE_FILTER_VALUE,
+  TLS_FILTER_VALUE,
+  SA_TOKEN_FILTER_VALUE,
+  OPAQUE_FILTER_VALUE
+];
+
+export const secretTypeFilterReducer = secret => {
+  switch (secret.type) {
+    case SecretType.dockercfg:
+    case SecretType.dockerconfigjson:
+      return IMAGE_FILTER_VALUE;
+
```

Comment:
Not sure it the extra line is on purpose, or I've missed it in the first round of review.

Reply:
It's on purpose. I think it makes it easier to visually group them since we switched to the enum

Reply:
Makes sense ! :+1: